### PR TITLE
Add maxRoutine and retainWindows flags to helm chart

### PR DESCRIPTION
**Pull request by tkerdoncuff**

### CODE REVIEW

File: helm/influxdb-athena-crawler/values.yaml [hunk 1]
```diff
@@ -24,6 +24,12 @@ defaults:
   # -- After how long to delete the objects.
   maxObjectAge: 5m
 
+  # -- Wether to retain this number of most recent folders
```
Comment (antonincms) [inline helm/influxdb-athena-crawler/values.yaml:27]:
```suggestion
  # -- Whether to retain this number of most recent folders
```

File: helm/influxdb-athena-crawler/values.yaml [hunk 2]
```diff
@@ -24,6 +24,12 @@ defaults:
   # -- After how long to delete the objects.
   maxObjectAge: 5m
 
+  # -- Whether to retain this number of most recent folders
+  retainWindows: ""
```
Comment (antonincms) [inline helm/influxdb-athena-crawler/values.yaml:28]: What is the expected format, true/false or is it a duration ?

Comment (tkerdoncuff) [reply helm/influxdb-athena-crawler/values.yaml:28]: It is a folder count, the whether in the comment was indeed confusing, is it better ?

Comment (antonincms) [reply helm/influxdb-athena-crawler/values.yaml:28]: Yes !

**Review by antonincms — APPROVED** [review]

**Pull request merged**